.github/workflows/test.yaml

```diff
@@ -5,7 +5,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go: ["1.19", "1.20" ]
+        go: ["1.22"]
```

Comment:
we should also test against `1.22` as that's still supported by go.

After that squash and push one commit, instead of `chore: ...` do `(misc) Update golang`

Reply:
Done :eyes: